src/components/asset-list/RecyclerAssetList2/profile-header/ProfileActionButtonsRow.tsx

```diff
@@ -0,0 +1,419 @@
+    if (!isReadOnlyWallet || enableActionsOnReadOnlyWallet) {
+      analytics.track('Tapped "Send"', {
+        category: 'home screen',
+      });
+
+      navigate(Routes.SEND_FLOW);
+    } else {
+      watchingAlert();
+    }
+  }, [navigate, isReadOnlyWallet]);
+
+  return (
+    <ActionButton icon="􀈟" onPress={handlePress} testID="send-button">
+      {lang.t('button.send')}
+    </ActionButton>
+  );
+}
+
+function MoreButton() {
+  // ////////////////////////////////////////////////////
+  // Handlers
+
+  const { accountAddress } = useAccountProfile();
+  const { navigate } = useNavigation();
+  const { isDamaged } = useWallets();
+
+  const handlePressAddCash = React.useCallback(() => {
+    if (isDamaged) {
+      showWalletErrorAlert();
+      return;
+    }
+
+    analytics.track('Tapped "Add Cash"', {
+      category: 'home screen',
+    });
+
+    if (ios) {
+      navigate(Routes.ADD_CASH_FLOW);
+    } else {
+      navigate(Routes.WYRE_WEBVIEW_NAVIGATOR, {
+        params: {
+          address: accountAddress,
+        },
+        screen: Routes.WYRE_WEBVIEW,
+      });
+    }
+  }, [accountAddress, isDamaged, navigate]);
+
+  const handlePressQRCode = React.useCallback(() => {
+    analytics.track('Tapped "My QR Code"', {
+      category: 'home screen',
+    });
+
+    navigate(Routes.RECEIVE_MODAL);
+  }, [navigate]);
+
+  // ////////////////////////////////////////////////////
+  // Action Sheet (iOS) / Context Menu (Android)
+
+  const { network } = useAccountSettings();
+
+  const isAddCashAvailable =
+    IS_TESTING === 'true' || network === NetworkTypes.mainnet;
+
+  const items = React.useMemo(
+    () => ({
+      addCash: lang.t('button.add_cash'),
+      myQRCode: lang.t('button.my_qr_code'),
+      cancel: lang.t('button.cancel'),
+    }),
+    []
+  );
+  const options = [
+    isAddCashAvailable ? items.addCash : null,
+    items.myQRCode,
+    ios ? items.cancel : null,
+  ].filter(x => x);
+
+  const ContextMenuButton = ios ? React.Fragment : ContextMenu;
+
+  const menuConfig = React.useMemo(
+    () => ({
+      menuItems: options.map((label, i) => ({
+        actionKey: i.toString(),
+        actionTitle: label,
+      })),
+    }),
+    [options]
+  );
+
+  const handlePressMenuItem = React.useCallback(
+    index => {
+      if (options[index] === items.addCash) {
+        handlePressAddCash();
+      }
+      if (options[index] === items.myQRCode) {
+        handlePressQRCode();
+      }
+    },
+    [
+      handlePressAddCash,
+      handlePressQRCode,
+      items.addCash,
+      items.myQRCode,
+      options,
+    ]
+  );
+
+  const handlePressContextMenuItem = useLatestCallback((e: any) => {
+    handlePressMenuItem(e.nativeEvent.actionKey);
+  });
+
+  const showActionSheet = () => {
+    showActionSheetWithOptions(
+      { options, cancelButtonIndex: options.length - 1 },
+      handlePressMenuItem
+    );
+  };
+
+  return (
+    <ContextMenuButton
+      // @ts-expect-error – JS component
```

Comment:
you can fix this I guess by doing 

`const ContextMenuButton = (ios ? React.Fragment : ContextMenu) as typeof ContextMenu`